### DISCUSSION TITLE
How to get YOLO achievement ?

### DISCUSSION THREAD

**Discussion by rahmatsubandi** [community/community#19146] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, how do I get the YOLO achievement? I've tried to do a pull request but can't either. What exactly is meant by without **code review**? can you give me an example?

**Comment by melvincwng** [comment, marked as answer]
Hj @rahmatsubandi, submit a PR in a public repository and just either a) directly merge it or b) add a friend as a reviewer but merge it without your friend approving the PR.

For more info, please click [here](https://github.com/Schweinepriester/github-profile-achievements).

Hope this helps.

**Reply by melvincwng** [reply]
@rahmatsubandi, if this reply has helped you, please kindly 'Mark as answer' so that others know this issue has been addressed.

Thank you and have a great day 😊

**Reply by AnzhiZhang** [reply]
It works for me, but it looks like a irresponsible merge achievement.

**Reply by melvincwng** [reply]
@AnzhiZhang, I wouldn't be too concern about the YOLO achievement tbh.

In the real working world, most companies would require your PR to be peer reviewed before it can be merged into the codebase.

Just enjoy your shiny new badge :D, and keep on coding!

**Reply by rahmatsubandi** [reply]
Hi, thank you for answering, tomorrow I will try if it works or not, I will 'Mark as Answer' your answer if it works.  Thanks buddy for the reference

**Reply by melvincwng** [reply]
@rahmatsubandi, pleasure to be able to assist you 😃. Hope you get your new shiny badge!

**Reply by rahmatsubandi** [reply]
Thanks, it's work! If I delete the repo where I got the badge, will it disappear too?

**Reply by rahmatsubandi** [reply]
Turns out I did and I didn't lose the badge

**Reply by AnzhiZhang** [reply]
> Thanks, it's work! If I delete the repo where I got the badge, will it disappear too?

No, it will not. You can see the repo name on your achievement, if you delete it, the repo name will be `inaccessible`
![image](https://user-images.githubusercontent.com/37402126/176106482-49d09726-925e-4f2c-8a82-f9e0516b7f29.png)

**Reply by rahmatsubandi** [reply]
> > Thanks, it's work! If I delete the repo where I got the badge, will it disappear too?
> 
> No, it will not. You can see the repo name on your achievement, if you delete it, the repo name will be `inaccessible` ![image](https://user-images.githubusercontent.com/37402126/176106482-49d09726-925e-4f2c-8a82-f9e0516b7f29.png)

Yes, it's still possible mate

**Reply by melvincwng** [reply]
@rahmatsubandi, thanks for marking this as answer. Enjoy the shiny YOLO badge on your profile :)

**Reply by Glazelf** [reply]
Please refrain from tagging people who aren't even participants in this discussion to beg for follows and opportunities to answer more discussions. @melvincwng 
Also; question was answered and this reply was offtopic anyways.

**Reply by sideedgetech** [reply]
It works bro

**Reply by GiMMC** [reply]
Create a repo

**Reply by rdrll** [reply]
Will the badge disappear if the repo got changed into a private repo?

**Reply by rodukov** [reply]
> Will the badge disappear if the repo got changed into a private repo?

No, it won't disappear. However, the link to the repository will become **inaccessible** to other people.

**Reply by joaoallmeida** [reply]
thanks bro!

**Reply by SonghanWu** [reply]
Could you explain more what does 'directly merge it' mean? Do I have to do it through terminal? Now everytime I merge in Github under 'Pull Request' it failed to give me the achievement, thank you!

**Comment by pablobion** [comment]
@rahmatsubandi  to get YOLO badge, you need to merge a PR directly into a public code repo without review.

**Reply by RobertGoodman08** [reply]
to get the YOLO badge, you need to merge the PR directly with the public code repository without verification.

**Comment by bossoff** [comment]
@rahmatsubandi for you to get YOLO achievement, you will have to always check for your Dependabot alert (We found potential security vulnerabilities in your dependencies.) in your repository for pull/marge dependency issues under security.

How to do that for the steps below:
![1](https://user-images.githubusercontent.com/30760187/175550871-615a3d41-334c-41f6-b22f-c0a65fe277b5.png)
![2](https://user-images.githubusercontent.com/30760187/175552115-653abd82-66f1-4c95-9d8f-2f3d2548b480.png)
![3](https://user-images.githubusercontent.com/30760187/175552468-5be608f7-acaa-4762-a4bf-d49c04af9849.png)
![4](https://user-images.githubusercontent.com/30760187/175552934-49846e82-aba7-4b3c-b0f2-137c9a486d97.png)
![5](https://user-images.githubusercontent.com/30760187/175554880-e028ad12-1c06-4db0-83d4-0c36574167ce.png)
![6](https://user-images.githubusercontent.com/30760187/175554926-0b1ba4cd-d314-427b-a9da-d46fe459d5b8.png)
![7](https://user-images.githubusercontent.com/30760187/175554941-59e807d7-08f2-4ec8-b1d7-3be63f1f1284.png)
![8](https://user-images.githubusercontent.com/30760187/175554975-947cfd8c-b90a-45bf-a15b-16a60d585a25.png)

Following the step is the only fastest way to get you the YOLO badge, it is called (You want it, You merge it). Good luck

**Reply by AnzhiZhang** [reply]
I created an empty repo and used @melvincwng's answer, so dependent bot will not be a problem.

EDIT: I mean that you do not need a complex repo, just create an empty repo, it will help you to reach this achievement!

**Reply by bossoff** [reply]
ooh cool. Here are some quick tips, try to download some frameworks e.g Laravel, Codeigniter, dart, or any kind of stack you prefer on your local machine PC then push it to that empty repo. In less than 3days I believe, you see dependency update notifications, you can now go from there.
https://laravel.com/
https://codeigniter.com/

**Reply by rilysh** [reply]
@bossoff  Github won't give YOLO badge if the account belongs a bot, only user account without a review, will work.

**Reply by rahmatsubandi** [reply]
Thank you for your review! Very good. But unfortunately I'm not working

**Reply by SenseiRofu** [reply]
@bossoff Thank you so much m8.

**Comment by rodukov** [comment]
Working way. You just need to:
1) Open your repository;
2) Create new branch;
3) Do some updates;
4) Then, create pull request to main branch;
5) **IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer;**
6) After this, just click merge as you do it always
7) Check your profile and see yolo acachievement.
* _If I helped you, make my answer the main one, I will be pleased._

**Reply by Owen-Choi** [reply]
thanks!

**Reply by vunguyenhuychuong** [reply]
thank you

**Reply by SenseiRofu** [reply]
@rodukov  ty so much m8

**Reply by TopTecShare** [reply]
Oh, so cool, your important statement solved my issue.
Love you!

**Reply by yusuftemel** [reply]
How can I add a reviewer to my repository?  [rodukov](https://github.com/rodukov)

**Reply by WendaoLee** [reply]
> How can I add a reviewer to my repository?  [rodukov](https://github.com/rodukov)

In your repository setting, invite someone to assist your manage that repository.So that you can add reviewer in pr page.

**Reply by naveen701526** [reply]
This trick is not working.

**Reply by CandyDEV1014** [reply]
This is not working for me

**Reply by udaypalecha** [reply]
Can it be done by adding myself as a reviewer?

**Reply by baohoangstan** [reply]
Thanks, it worked for me

**Comment by GeneralChauhan** [comment]
Hj @rahmatsubandi, you need to submit a PR in a public repository merge it without your the owner approving the PR.

**Reply by vunguyenhuychuong** [reply]
i got it!!!

**Reply by CandyDEV1014** [reply]
Thanks

**Comment by RobertGoodman08** [comment]
to get the YOLO badge, you need to merge the PR directly with the public code repository without verification.

**Reply by CandyDEV1014** [reply]
thanks for your time

**Comment by cardison** [comment]
1. You should install github for dekstop (because this is the easiest way)
    2.Open any repository on your github desktop and clone it to local storage on your computer
    3.After that, create a new brach to commit
   4. Make a change in the new branch, then you commit by including CO-AUTHOR in the commit message column (you can add me to CO-AUTHOR)
    5.Then do a pull request as usual and merge it. And taaadaa you will get the achievement

**Comment by baraaj** [comment]
Good evening,how can i get YOLO achievement please?

**Reply by yusuftemel** [reply]
Good evening, create a repository
Then create a new branch in it.
Modify this repo( make some changes in it)
Add a reviewer( first you should give an access to your friend in settings)
Merge request and all is done!
Good luck.
Can you sign my reply as answered, please?

**Reply by baraaj** [reply]
thank you

**Reply by yusuftemel** [reply]
You didn't mark it as an answer, can you mark me as well, please? @baraaj

**Reply by baraaj** [reply]
Done @yusuftemel

**Comment by thclark** [comment]
Try not to get it. It's not something to be proud of, it's bad practice (although unavoidable if you're a sole maintainer).

**Reply by karendolan** [reply]
I had a fully reviewed pull, back and forth conversations. They approved it but did not click the green approve button, I merged it after they approved it, but the green approve button was not clicked. And now I have that embarrassing YOLO label! Do you know who I can appeal to to request its removal? It's a mark of shame, that I didn't earn

**Reply by rilysh** [reply]
> I had a fully reviewed pull, back and forth conversations. They approved it but did not click the green approve button, I merged it after they approved it, but the green approve button was not clicked. And now I have that embarrassing YOLO label! Do you know who I can appeal to to request its removal? It's a mark of shame, that I didn't earn

The one and only way you can get rid from this is to uncheck "Show Achievements" in your profile settings. I understood you, in my case I recently merged my own PR without reviewing, I feel like I did that because of my cretinism as a result I got that YOLO badge.
In the end, GitHub is a proprietary platform, last thing we can do "beg" to them everything else is controlled by them, sadly.

**Comment by phuccoder2000** [comment]
to get the YOLO badge, you need to merge the PR directly with the public code repository without verification.

**Reply by karendolan** [reply]
What does "verification" mean? The way my colleague and I got our YOLO badges was to have a thorough peer review of each other's code on it's build platform. In the multiple source branches. Lots of improvements, back and forth and testing. But without the click on github pull's sanctioned approve button. Because we communicate all the time about our code but not always asynchronously through github. The badge shows a link to the giant pull with "Merged without a review" text next to it, which is kind of funny, totally inaccurate. It would be better for the text to be "Merged without someone clicking github's approve button". I used to like the github tags, I thought they were fun. I'm going to uncheck the show achievements button, because these achievements don't represent anything about the quality and dedication of contributing to shared projects.

**Comment by noviicee** [comment]
I have done the same many times, but did not receive YOLO. Why is that so? :thinking:

**Reply by noviicee** [reply]
> I have done the same many times, but did not receive YOLO. Why is that so? thinking

E.g:- [This one](https://github.com/noviicee/Banque-De-Questions/pull/34)
[And this too](https://github.com/noviicee/Banque-De-Questions/pull/33)
*(from what I could find from my most recent)*
....and many more!

**Reply by Atuoha** [reply]
The first one wasn't approved. Try it out with public repo of a friend or...

**Reply by noviicee** [reply]
I am thinking that maybe I don't want it now.
I read that it's kind of a bad practice :)

**Reply by Jack-tendy-538** [reply]
me too
[this](https://github.com/Jack-tendy-538/Rubik_cube-jack.s_edition/pull/4) and laughably [this](https://github.com/Jack-tendy-538/countdown/pull/2) (I merged before dependency review comments :laughing: )

**Comment by GiMMC** [comment]
create a repo without review

**Reply by GiMMC** [reply]
Thank you

**Comment by jso4342** [comment]
You need tp submit a PR in a public repository (just invite your friend to give an access to her/him) 
and directly merge it and add a friend as a reviewer.
Now you just have to merge it without your friend approving the PR.

**Comment by Pedrokaruna9** [comment]
Ultimately, the idea of Bitcoin is elegant. One twenty-one millionth of all of the value on the network for as long as the network may last. There’s nothing more stable, nothing more predictable in the entire financial universe than that. That is the single most stable body in the entire financial universe. If you’re looking for something to revolve around there is nothing more stable.

Michael Saylor on Bitcoin

**Comment by minhbtrc** [comment]
Good evening, how can i get YOLO achievement please?

**Comment by RizWaaN3024** [comment]
Create the pull request: Open the pull request as usual, whether from the GitHub web interface or using a command line.

Skip code review: Ensure no code review is performed on the pull request. GitHub won’t mark the pull request as reviewed if no one leaves any review comments or approvals.

Merge without reviewing:

If you are using the GitHub web interface, click on the green "Merge pull request" button without adding any review comments.
If your repository allows merging without a review, you can directly merge the PR without waiting for anyone else to approve or reject it.

**Comment by etsibeko-dev** [comment]
Have a public repo. In your repo you'll have a main, master or whatever branch. Create a 2nd branch, edit one file. Just a minor tweak like replace a certain word in the Readme with a synonym. After that, save, compare, and create a PR on the main branch - the default one. Then just merge the PR without a review.

BTW, I know you already the badge and the question was years ago. But, to anyone who will come across this answer. Just know that I did what I said above, and got my YOLO badge immediately.

_Happy collaboration. ❤️_
